### ISSUE TITLE
error

### ISSUE THREAD

**Issue by EddieMorra** [avocode/avocode#1752]
---
* **Version**: 2.17.0
* **Web/Desktop**: Desktop
* **Platform**: win32
* **Avocode UID**: 217805
---
```
TypeError: Cannot read property 'toArray' of undefined
    at Function.value (file:///C:/Users/roma/AppData/Local/avocode/app-2.17.0/resources/app.asar/dist/avocode.min.js:1:141306)
    at file:///C:/Users/roma/AppData/Local/avocode/app-2.17.0/resources/app.asar/dist/avocode.min.js:1:140994
    at List.__iterate (C:\Users\roma\AppData\Local\avocode\app-2.17.0\resources\app.asar\node_modules\immutable\dist\immutable.js:2208:13)
    at List.forEach (C:\Users\roma\AppData\Local\avocode\app-2.17.0\resources\app.asar\node_modules\immutable\dist\immutable.js:4383:19)
    at Function.value (file:///C:/Users/roma/AppData/Local/avocode/app-2.17.0/resources/app.asar/dist/avocode.min.js:1:140927)
    at e.value (file:///C:/Users/roma/AppData/Local/avocode/app-2.17.0/resources/app.asar/dist/avocode.min.js:1:2966965)
    at r._handleCopySvg (file:///C:/Users/roma/AppData/Local/avocode/app-2.17.0/resources/app.asar/dist/avocode.min.js:1:1349149)
    at CallbacksRegistry.apply (C:\Users\roma\AppData\
```

**Comment by jakubzitny** [comment]
Hi @EddieMorra, thanks for reporting this. This is a bug and we'll try to fix it as soon as possible.

**Comment by jakubzitny** [comment]
Hi @EddieMorra, this should be fixed in latest Avocode 2.20.0. Please update and let us know if you have any other problems 😉
